Rom.py

```diff
@@ -179,4 +191,4 @@ def patch_rom(world, rom):
         rom.write_byte(0x20811A5, 0x63)
         rom.write_bytes(0x20811B2, [0xFF, 0x01])
 
-    return rom
\ No newline at end of file
+    return rom
```

Comment:
I don't know why this changed, I didn't touch it.. It may be because I used vim to edit it